04_numeronym/NimishMudhale/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronym(vals ...string) []string {
+	numeronymArray := []string{}
```

Comment:
This is fine, but if you really care about runtime you could optimise by preallocating the size of `numeronyms`:

```go
numeronymArray := make([]string, len(vals))
for i, val := range vals {
		numeronymArray[i] =  numeronym(val)
}
```